hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/processors/CustomMessageCallProcessorTest.java

```diff
@@ -171,7 +174,7 @@ void valueCannotBeTransferredToSystemContracts() {
         givenHaltableFrame(isHalted);
         givenCallWithCode(ADDRESS_6);
         given(addressChecks.isSystemAccount(ADDRESS_6)).willReturn(true);
-        given(registry.get(ADDRESS_6)).willReturn(nativePrecompile);
+        lenient().when(registry.get(ADDRESS_6)).thenReturn(nativePrecompile);
```

Comment:
Why do we want to use lenient() instead of just removing the stubbing?

Reply:
It seems like it was necessary before because `(precompiles.get(codeAddress) == null)` was getting called in `doHaltIfInvalidSystemCall`. But this is no longer the case.

Reply:
Agreed, it is still called though but we reversed the statements so its not needed, yup